src/main/java/com/microsoft/sqlserver/jdbc/DDC.java

```diff
@@ -614,9 +614,9 @@ private static LocalDateTime parseStringIntoLDT(String s) {
         if (firstDash > 0 && secondDash > 0 && secondDash < dividingSpace - 1) {
             if (firstDash == YEAR_LENGTH && (secondDash - firstDash > 1 && secondDash - firstDash <= MONTH_LENGTH + 1)
                     && (dividingSpace - secondDash > 1 && dividingSpace - secondDash <= DAY_LENGTH + 1)) {
-                year = Integer.parseInt(s, 0, firstDash, 10);
-                month = Integer.parseInt(s, firstDash + 1, secondDash, 10);
-                day = Integer.parseInt(s, secondDash + 1, dividingSpace, 10);
+                year = Integer.parseInt(s.subSequence(0, firstDash).toString());
```

Comment:
Is there a reason why we're using `subSequence(int,int)` here instead of `subString(int,int)`. It seems like `subSequence` actually uses `subString` internally and the only major difference is that it returns a `CharSequence` instead of a `String`. Considering it seems we want a `String` anyways, lets just use `subString(int,int)`.